src/Sarif/RuntimeConditions.cs

```diff
@@ -49,12 +49,19 @@ public enum RuntimeConditions : uint
         ExceptionInstantiatingSkimmers = 0x8000,
 
         // Non-fatal conditions
+        RuleWasExplicitlyDisabled = 0x04000000,
         RuleCannotRunOnPlatform   = 0x08000000,
         RuleNotApplicableToTarget = 0x10000000,
         TargetNotValidToAnalyze   = 0x20000000,
         OneOrMoreWarningsFired    = 0x40000000,
         OneOrMoreErrorsFired      = 0x80000000,
 
-        Nonfatal = 0xF8000000
+        // 0xFC000000
```

Comment:
We'll need to update our BinSkim documentation.

We also should consider if we just want 0xFFC or similar to be our 'non-fatal exit code' space, since when we change this it may affect teams integrating the tool.